cli_tools/gce_windows_upgrade/upgrade_script_2008r2_to_2012r2.ps1

```diff
@@ -21,13 +21,13 @@ try {
   Remove-Item 'C:\$WINDOWS.~BT' -Recurse -ErrorAction Ignore
 
   $ver=[System.Environment]::OSVersion.Version
-  Write-Host "windows_upgrade_current_version=$($ver.Major).$($ver.Minor)"
   if ("$($ver.Major).$($ver.Minor)" -ne "6.1") {
     if ("$($ver.Major).$($ver.Minor)" -eq "6.3") {
       Write-Host "GCEMetadataScripts: The instance is already running Windows 2012R2!"
-      Return
+      Write-Host "GCEMetadataScripts: Rerunning upgrade.ps1 for post-upgrade step..."
+    } else {
+      throw "The instance is not running Windows 2008R2!"
```

Comment:
A couple of recommendations:
1. Remove the exclamation point
2. Include the detected version
3. Clarify which instance -- I'm assuming it means the base version, but in isolation it could mean that upgrade occurred and you didn't reach the target version

Reply:
Done for #1 and #2. For #3, we don't know what is the reason. It rarely happens since the param-validation stage could have found this issue.